float32 pan_angle
float32 tilt_angle
bool wait_finished
---
float32 pan_angle
float32 tilt_angle

